spec/requests/dashboard_lists_spec.rb

```diff
@@ -58,107 +58,6 @@
 
         expect(response.body).not_to include('Jane Doe')
       end
-
```

Comment:
This behaviour is now specified in spec/system/dashboard/viewing_in_progress_applications_spec.rb